src/docs/command-line.dox

```diff
@@ -413,7 +413,7 @@ The \c verify command verifies the specified table, exiting success if
 the data source is correct, and failure if the data source is corrupted.
 
 @subsection util_verify_synopsis Synopsis
-<code>wt [-RrVv] [-C config] [-E secretkey ] [-h directory] verify [-d dump_address | dump_blocks | dump_history | dump_layout | dump_offsets=#,# | dump_pages ] [-S] uri</code>
+<code>wt [-RrVv] [-C config] [-E secretkey ] [-h directory] verify [-d dump_address | dump_blocks | dump_history | dump_layout | dump_offsets=#,# | dump_pages ] [-h] [-S] uri</code>
```

Comment:
"uri" is optional with "-h". To be further discussed in WT-5556.

Reply:
So Should I keep the usage documentation as I have it, despite it's confusing nature?